src/lib.rs

```diff
@@ -260,6 +596,8 @@ impl UTransport for ULinkZenoh {
 
     async fn unregister_listener(&self, topic: UUri, listener: &str) -> Result<(), UStatus> {
         // Do the validation
+        // TODO: Whether we should check URI is resolved or not
```

Comment:
According to latest discussion, `ULinkZenoh` should only use the `Uuri` in micro form without requiring any validation.

Reply:
Use micro form now.